dspace-uclouvain/src/main/java/org/dspace/uclouvain/packager/DSpaceUCLouvainMETSIngester.java

```diff
@@ -2,23 +2,68 @@
 
 import org.apache.logging.log4j.Logger;
 import org.dspace.authorize.AuthorizeException;
+import org.dspace.content.Bitstream;
 import org.dspace.content.DSpaceObject;
 import org.dspace.content.Item;
+import org.dspace.content.MetadataField;
+import org.dspace.content.authority.Choices;
 import org.dspace.content.crosswalk.CrosswalkException;
 import org.dspace.content.crosswalk.MetadataValidationException;
-import org.dspace.content.packager.*;
+import org.dspace.content.factory.ContentServiceFactory;
+import org.dspace.content.packager.AbstractMETSIngester;
+import org.dspace.content.packager.DSpaceMETSIngester;
+import org.dspace.content.packager.METSManifest;
+import org.dspace.content.packager.PackageParameters;
+import org.dspace.content.packager.PackageValidationException;
+import org.dspace.content.service.BitstreamService;
+import org.dspace.content.service.MetadataFieldService;
 import org.dspace.core.Constants;
 import org.dspace.core.Context;
+import org.dspace.services.ConfigurationService;
+import org.dspace.services.factory.DSpaceServicesFactory;
 import org.dspace.workflow.WorkflowException;
+import org.jdom2.Content;
+import org.jdom2.Element;
+import org.jdom2.transform.JDOMResult;
+import org.jdom2.transform.JDOMSource;
 
+import javax.xml.transform.*;
```

Comment:
Could be change to literal imports like `javax.xml.trasnform.Transformer` for example.
Actually, it is not that big of a deal since there are only 12 classes in this package and you are using 6 of them so ...

Reply:
Actually, this is not important, it seems like the compiler is only keeping used classes in the final `.class` file:
![image](https://github.com/bibsys/DSpace/assets/71372371/f96fbeda-b0a2-491b-9ffe-0e32e87b7257)